api/filter/accesslog.proto

```diff
@@ -9,24 +9,53 @@ import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
-message AccessLogEntry {
-  // The HTTP request method (RFC 7231/2616)
-  RequestMethod request_method = 1;
+// Envoy access logs describe incoming interaction with Envoy over a fixed
+// period of time, and typically cover a single request/response exchange.
+// Except where explicitly declared otherwise, all fields describe
+// *downstream* interaction between Envoy and a connected client.
+// Fields describing *upstream* interaction are prefixed with the string
+// `upstream_`.
+message AccessLogBaseEntry {
+  // This field indicates the rate at which this log entry was sampled.
+  // Valid range is (0.0, 1.0].
+  double sample_rate = 1;
+
+  // This field is the IP and port on which the request from the user was
+  // received, stored in ipv4:port or [ipv6]:port format.
+  string destination_host = 2;
+
+  // Whether the request arrived via a secure (TLS) protocol
+  google.protobuf.BoolValue secure = 3;
+
+  // Properties of the TLS connection, if used
+  TLSProperties tls_properties = 4;
 
   // The time that Envoy started servicing this request
-  google.protobuf.Timestamp start_time = 2;
+  google.protobuf.Timestamp start_time = 7;
```

Comment:
nit: Can we reorder fields as long as we are doing this since we are not wire locked yet.

Reply:
yeah already noticed and fixed in update I'm pushing today